worlds/tracker/TrackerClient.py

```diff
@@ -150,6 +150,24 @@ def _cmd_reset_ignored(self):
         self.ctx.ignored_locations.clear()
         updateTracker(self.ctx)
         logger.info("Reset ignored locations.")
+    def _cmd_next_progression(self):
+        """Finds all items that will unlock a check immediately when collected, and a best guess of how many new checks they will unlock."""
+        updateTracker(self.ctx)
+        baseLocs = len(self.ctx.locations_available)
+        counter = Counter()
+        items_to_check = {item.name for item in self.ctx.multiworld.get_items() if item.game == self.ctx.game and ItemClassification.progression in item.classification} 
```

Comment:
Changes to use the advancement property, also changes to check player matching instead of game to avoid false positives with items that don't exist
```suggestion
        items_to_check = {item.name for item in self.ctx.multiworld.get_items() if item.player == self.ctx.player_id and item.advancement}
```